### ISSUE TITLE
Suggestion: Only remind to boost after x amount of time for skill drain

### ISSUE THREAD

**Issue by Koslas1** [JonneSaloranta/skilling-boost-reminder#6]
What would be the chance of setting in the config showing the reminder after x amount of seconds since your skill has drained (With 0 set as default meaning anytime)

In theory if I set that to 10, it would only remind me to boost, if my special attack is restored, and it's been 10 seconds or less than my skill boost draining. That way I won't feel like I'm wasting 1 level, as the plugin is suggesting me to do here 
![image](https://github.com/user-attachments/assets/6240bec3-635f-4e36-b083-13263f0ed504)

It is suggesting I boost with 11 seconds remaining on my current, therefore in 11 seconds after boosting, I would lose 1 level of the boosted skill, instead of waiting for 11 seconds before reminding me to boost.
This probably wouldn't matter the 1st time you boost when you don't know when skill boosts will drain, as the game is weird, and what I've found is that if it's a new boost, it won't drain within a certain timeframe (Meaning if you skill is meant to drain in 5 seconds, it's in the grace period, and your skill drain timer would go back to a minute without draining), however when it's boosting something existing, such as a currently +1 woodcutting, to +3 woodcuting, even if your boost will drain on the next tick, you would then go back to a +2 boost

**Comment by Koslas1** [comment]
Here is an example of what I want to try an avoid 

https://github.com/user-attachments/assets/7f6ee724-7944-4865-aefc-81d52c3b7b78

**Comment by JonneSaloranta** [comment]
Yeah good point. Maybe I could make the notification happen only after the 'next - restore' or with override setting to notify every time regardless of the next restore.

Settings:
- Notify after 'next - restore' (Defaults to True)

Is there anything else that should be added?

**Comment by Koslas1** [comment]
Not that I can think of, however I do have a bug, which I will make a new issue for just to keep bugs and suggestions separate
